### PR TITLE
Actual implementation

### PR DESCRIPTION
**Pull request by PyroTechniac**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import { performance } from 'perf_hooks';
+
+/**
+ * Klasa's Stopwatch class, uses native node to replicate/extend performance now dependency.
+ */
+export default class Stopwatch {
```
Comment (vladfrangu) [inline src/index.ts]: Shouldn't you export the class, not as default?

Exporting as default means people will have to do `const { default: Stopwatch } = require('@klasa/stopwatch')` in commonJS

Comment (PyroTechniac) [reply src/index.ts]: I believe it's what was requested in another repo, `Type` if I recall correctly

Comment (PyroTechniac) [reply src/index.ts]: Also, all of these utils will be reexported by klasa itself, so end users can just require it like they have been

Comment (vladfrangu) [reply src/index.ts]: https://github.com/dirigeants/type/blob/f34fc151169c425f17acdc6ede14527c6c9ac945/src/index.ts#L8

**Pull request merged**